lib/omnibus/config.rb

```diff
@@ -333,6 +333,13 @@ def reset!
     # @return [Integer]
     default(:publish_retries, 2)
 
+    # Directory pattern for publisher. Interpolation of matadata keys are supported.
+    #
+    # @example 'custom_prefix/%{platform}/%{platform_version}/%{arch}/%{basename}'
+    #
+    # @return [Integer]
+    default(:publish_dir_pattern, nil)
```

Comment:
Would it make sense to set the default **here** instead of [having it in this line?](https://github.com/chef/omnibus/pull/790/files#diff-eee644ad5e745ef006b9318510bacb7fR271)